### ISSUE TITLE
Unable to hide the extensions window

### ISSUE THREAD

**Issue by ryancole** [Microsoft/vscode#23682]
I have the activity pane disabled. I've opened the extensions window, via `CTRL+SHIFT+X`. I'm now unable to close the extensions window.

Here's a screenshot of what it looks like ...

![untitled](https://cloud.githubusercontent.com/assets/484801/24514455/fb5ae83e-1539-11e7-8379-07d8529a0ab0.png)

```
Version 1.11.0-insider
Commit bc9a7bb1f90557c271c39628b5a3cdd7dfbc750c
Date 2017-03-30T08:11:58.005Z
Shell 1.4.6
Renderer 53.0.2785.143
Node 6.5.0
```

**Comment by Tekbr** [comment]
@ryancole Try <kbd>Ctrl</kbd>+<kbd>b</kbd>

**Comment by ryancole** [comment]
@Tekbr that does close the extensions window, but also closes the file explorer window.

**Comment by bpasero** [comment]
I think you are asking for something that does not exist: views in the sidebar are not overlapping each other, so you cannot close them. You can either hide the sidebar or make another view active. 

Can you rephrase your request, do you expect a tabs like behaviour in the side bar?

**Comment by ryancole** [comment]
@bpasero I think you're correct. I was using ctrl+shift+b to bring up the extensions pane. I thought it was overlapping the file explorer pane, but, as you just informed me, it's not actually overlapping. So, with that info, I now understand what it's doing and how to use it. I can bring back the file explorer pane with ctrl+shift+e.
